packages/lit-element-starter-js/custom-elements.json

```diff
@@ -0,0 +1,70 @@
+{
```

Comment:
shouldn't this not be checked in? Is it in the original project?

Reply:
Yeah, it's checked in in the original templates: https://github.com/PolymerLabs/lit-element-starter-ts/blob/master/custom-elements.json. Not sure what the thinking was there...? The generated docs are also checked in, but that's to use github pages's easy sub-folder publishing setting right?

Reply:
Yeah, generated docs are checked in for github pages. I don't think this should be there since it's not used in the static site.